Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Dwarven Portraits

## Changelog:
- Added Frostborn, Wildhammer, Bronzebeard and Dark Iron dwarven portraits together with the undead and fel versions.


# How to test:
`event WCPOD.5013` to become an undead.
`event WCFEL.9950` to spam Fel Corruption and become a demon.

After this, try to invite courtiers, check if they have racial traits.
